src/lib.rs

```diff
@@ -46,6 +67,200 @@ impl UniffiCustomTypeConverter for Uuid {
     }
 }
 
+pub struct StatusLists;
```

Comment:
Could you put the status list related code, and the vc-barcode code, in their own files? I thought it wasn't possible with uniffi but it actually is if you use `pub mod`